daemon/logger/syslog/syslog.go

```diff
@@ -25,9 +28,40 @@ func init() {
 	}
 }
 
+func parseConnectionUrl(c map[string]string) (string, string, error) {
+	rawurl, _ := c["syslog-connection-url"]
```

Comment:
other drivers may have their own parsing routine to decide if `address` is correct